native/src/components/PoiFiltersModal.tsx

```diff
@@ -125,7 +127,11 @@ const PoiFiltersModal = ({
           </TileRow>
         </Section>
         <Section>
-          <StyledTextButton onPress={closeModal} text={t('showPois')} />
+          {poisCount !== 0 ? (
+            <StyledTextButton onPress={closeModal} text={t('showPois', { count: poisCount })} />
+          ) : (
+            <StyledTextButton onPress={closeModal} text={t('noPois')} disabled />
+          )}
```

Comment:
The translation `showPois` you created now also handle the case of `poisCount === 0`, so no need to handle this separately here. Could you try to simplify this? Same for web :)

Reply:
alright